controllers/constant/constant.go

```diff
@@ -114,6 +114,11 @@ const (
 	CSMappingConfig = "ibm-cs-ns-mapping-webhook-configuration"
 )
 
+// Bedrock Deployment names for No OLM
+var (
+	DeploymentsName = []string{"ibm-common-service-operator", "operand-deployment-lifecycle-manager", "ibm-namespace-scope-operator", "ibm-commonui-operator", "ibm-iam-operator"}
```

Comment:
I don't see why we need to put status info on the operator deployments. OLM cs-operator did not give status on operators, so why do we need to do this for non-OLM mode?